Lib/test/test_peg_generator/test_c_parser.py

```diff
@@ -24,7 +26,8 @@ def setUp(self):
         self.tmp_path = tempfile.mkdtemp()
 
     def tearDown(self):
-        shutil.rmtree(self.tmp_path)
+        with contextlib.suppress(PermissionError):
```

Comment:
This is needed because extension modules that are loaded in the interpreter cannot be deleted on windows until the interpreter finishes.